Lib/ufo2ft/fontInfoData.py

```diff
@@ -123,21 +123,27 @@ def openTypeNameWWSSubfamilyNameFallback(info):
 
 def openTypeOS2TypoAscenderFallback(info):
     """
-    Fallback to *unitsPerEm * 1.2 + descender*.
+    Fallback to *ascender + typoLineGap*.
     """
-    return int(info.unitsPerEm * 1.2) + info.descender
+    return info.ascender + getAttrWithFallback(info, "openTypeOS2TypoLineGap")
```

Comment:
The `openTypeOS2TypoAscenderFallback` needs to be only `info.ascender`.

Applications that uses the OS/2 Typo metrics already add the OS/2 `TypoLineGap` to the `TypoAscender` and `TypoDescender` in order to calculate the total line spacing.

If you also add the `TypoLineGap` when calculating the `TypoAscender` fallback, then a non-zero line gap will end up being added twice.

For example:
```python
>>> from ufo2ft.fontInfoData import getAttrWithFallback
>>> class Info(object):
...     pass
>>> info = Info()
>>> info.unitsPerEm = 1000
>>> info.ascender = 700
>>> info.descender = -300
>>> typoLineGap = getAttrWithFallback(info, 'openTypeOS2TypoLineGap')
>>> typoAscender = getAttrWithFallback(info, 'openTypeOS2TypoAscender')
>>> typoDescender = getAttrWithFallback(info, 'openTypeOS2TypoDescender')
>>> typoAscender - typoDescender + typoLineGap
1400
>>> hheaLineGap = getAttrWithFallback(info, 'openTypeHheaLineGap')
>>> hheaAscender = getAttrWithFallback(info, 'openTypeHheaAscender')
>>> hheaDescender = getAttrWithFallback(info, 'openTypeHheaDescender')
>>> hheaAscender - hheaDescender + hheaLineGap
1200